indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -37,4 +37,67 @@
 
         setupListGenerator(filterConfig);
     };
+
+    global.setupAbstractPage = function setupAbstractPage() {
+        $('body').on('click', '.judge-button', function(evt) {
+            var $box = $('#abstract-decision-box');
+            var $this = $(this);
+            var $form = $box.find('form');
+
+            evt.preventDefault();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                data: getParamsFromSelectors($form.find('input:not(:disabled), textarea, select:not(:disabled)')),
+                success: function(data) {
+                    if (data.page_html) {
+                        $('.management-page').replaceWith(data.page_html);
+                        $box = $('#abstract-decision-box');
+                    } else {
+                        $box.html(data.box_html);
+                    }
+                    initForms($box);
+                    showFormErrors($box);
+                }
+            });
+        }).on('click', '.review-button', function(evt) {
+            var $this = $(this);
+            var $box = $this.closest('.abstract-review-box');
+            var $form = $box.find('form');
+
+            evt.preventDefault();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                data: getParamsFromSelectors($form.find('input[type="text"]:not(:disabled), \
+                                                         input:checked, input[type="hidden"], \
+                                                         textarea, select:not(:disabled)')),
+                success: function(data) {
+                    if (data.page_html) {
+                        $('.management-page').replaceWith(data.page_html);
+                    } else {
+                        $box.replaceWith(data.box_html);
+                    }
+                    initForms($('.management-page'));
+                    showFormErrors($('.management-page'));
+                }
+            });
+        }).on('click', '.change-review-button', function(evt) {
+            evt.preventDefault();
+
+            var $reviewDisplay = $(this).closest('.i-timeline-item');
+            var $reviewBox = $reviewDisplay.next('.abstract-review-box');
+            $reviewDisplay.hide();
+            $reviewBox.show();
+        }).on('indico:htmlUpdated', function() {
+            initForms($(this));
+            showFormErrors($(this));
```

Comment:
Not a strong argument since it's only 2 lines, but on the other hand... it's 2 lines! :yum: 
```js
var $this = $(this);
```